data-validation/multiple-coordinate-schemas/README.md

```diff
@@ -203,3 +203,14 @@ To upload `policy.json` to the broker, run the following command:
 ```bash
 curl -X POST --data @policy.json -H "Content-Type: application/json" http://localhost:8888/api/v1/data-validation/policies
 ```
+
+## Quality Metric
+Check out the script `generate-random-data.sh` that continously generates random GPS coordinates according to the `usa-coordinates` schema.
+However, some of them are invalid. Since we added a metric function to count invalid and valid messages a quality metric can be easily defined.
+
+Suppose you have a Prometheus server running andPrometheus Extension installed available at [HiveMQ's website](https://www.hivemq.com/extension/prometheus-extension/) you can run a PromQL statement to derive a quality metric:
+
+```
+com_hivemq_data_governance_hub_data_validation_custom_counters_valid_coordinates / (com_hivemq_data_governance_hub_data_validation_custom_counters_invalid_coordinates + com_hivemq_data_governance_hub_data_validation_custom_counters_valid_coordinates)
+```
+Consequently, alerts can be defined to be notified for certain threshold violation.
```

Comment:
This sounds a bit unclear. Maybe something like "With this, it is now also possible to define alerts which notify when a certain threshold of invalid messages is reached".